### PR TITLE
Added GitLab integration

### PR DESCRIPTION
**Pull request by joetanx**
Changes:
✏️: Edited
➕: Added

.
│   README.md✏️
│
├───apps
│   └───gitlab
│       ├───aws-cli-demo
│       │       .gitlab-ci.yml➕
│       │       main.tf➕
│       │
│       ├───terraform-aws-s3-cleanup
│       │       .gitlab-ci.yml➕
│       │
│       └───terraform-aws-s3-demo
│               .gitlab-ci.yml➕
│               demo.txt➕
│               main.tf➕
│               provider.tf➕
│
└───policy
    │   conjur_load_policy.sh✏️
    │
    ├───conjur
    │   └───authn-jwt
    │       │   authn-jwt-gitlab.yml➕
    │       │
    │       └───gitlab
    │               grant-gitlab.yml^➕
    │
    └───data
            apps-gitlab.yml➕
            entitle-gitlab.yml➕

### CODE REVIEW

**Comment by quincycheng** [comment]
Thanks @joetanx !  Just got a minor comment and everything else looks fine.  

Very well done!

File: README.md
Comment (joetanx) [inline README.md:1]: test

**Review by joetanx — COMMENTED** [review]
test

File: policy/conjur_load_policy.sh
Comment (joetanx) [inline policy/conjur_load_policy.sh:1]: test

File: README.md
Comment (joetanx) [inline README.md:1]: test

**Pull request merged**